src/controllers/user/Controller.ts

```diff
@@ -0,0 +1,70 @@
+import {Request, Response} from 'express';
+class UserController{
```

Comment:
all same mistakes done in trainee module to be corrected here

Reply:
Done